packages/create-react-native-library/src/index.ts

```diff
@@ -356,6 +388,7 @@ async function create(argv: yargs.Arguments<any>) {
       cpp: languages === 'cpp',
       kotlin: languages === 'kotlin-objc' || languages === 'kotlin-swift',
       swift: languages === 'java-swift' || languages === 'kotlin-swift',
+      windows: languages === 'windows-cpp' || languages === 'windows-csharp',
```

Comment:
suggest that we call these uwp-cpp and uwp-csharp, or windows-uwp-cpp / csharp
We will eventually have non-UWP templates too (working on it as we speak), which are for win32 apps using WinUI 3 and the Windows App SDK (formerly known as Project Reunion)